src/test/conditions/PreimageSha256_test.cpp

```diff
@@ -333,6 +333,7 @@ class PreimageSha256_test : public beast::unit_test::suite
                 log << "Fulfillment deserialize error: " << std::get<0>(x) << " " << ec.message() << '\n';
             }
             auto const c = Condition::deserialize (hexblob(std::get<2>(x)), ec);
```

Comment:
I'm confused about `c` here - do we care about the value? Should we verify correctness for it , or just ignore it?